### PR TITLE
Issue #2 - print version information.

### PR DESCRIPTION
**Pull request by ewheath**

### CODE REVIEW

**Comment by ewheath** [comment]
@dspasojevic Have a look at the basic setup when you get a chance.  Comments are welcome!

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+artifact in (Compile, assembly) := {
+  val art = (artifact in (Compile, assembly)).value
+  art.copy(`classifier` = Some("assembly"))
+}
+
+addArtifact(artifact in (Compile, assembly), assembly)
+
+// To publish, put credentials in ~/.ivy2/credentials
+credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
```
Comment (dspasojevic) [inline build.sbt]: This can be removed.

File: project/plugins.sbt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+resolvers += Classpaths.sbtPluginReleases
+
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.9.0")
```
Comment (dspasojevic) [inline project/plugins.sbt]: Remove.

File: src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala [hunk 3]
```diff
@@ -0,0 +1,55 @@
+package au.com.agiledigital.toolform.app
+
+import com.typesafe.config._
+
+/**
+  * The toolform app is a transformation tool that generates CI/CD pipelines from a high-level HOCON formated descriptor.
```
Comment (dspasojevic) [inline src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala]: Shorten as per comment on other PR.

File: src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    * Runs the tool.
+    *
+    * @param args Command line arguments.
+    * @return Either[String, Config] - The configuration or an error.
+    */
+  def execute(args: Array[String]): Either[String, Config] = {
+    val parser = new scopt.OptionParser[CommandConfig]("toolform") {
+      head("toolform", "0.1")
+      help("help").abbr("h").text("Displays this usage text.")
+      version("version").abbr("v").text("Displays the version text.")
```
Comment (dspasojevic) [inline src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala]: "Displays version information"?

File: src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      sys.exit(1)
+    case Right(config) =>
+      displayConfiguration(config)
+  }
+}
+
+/**
+  * The configuration for the tool
+  */
+case class CommandConfig() {
```
Comment (dspasojevic) [inline src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala]: `final case class`.

File: build.sbt [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import sbtassembly.AssemblyPlugin._
+
+name := "toolform"
+
+organization := "au.com.agiledigital"
+
+// This version number will be modified by the build system. It should be of the form x.y, not x.y.z
+version := "0.1"
+
+scalaVersion := "2.12.2"
```
Comment (dspasojevic) [inline build.sbt]: `2.12.3`

File: project/build.properties [hunk 7]
```diff
@@ -0,0 +1 @@
+sbt.version=0.13.8
```
Comment (dspasojevic) [inline project/build.properties]: Would prefer `1.x` if the plugins are compatible.

Comment (ewheath) [reply project/build.properties]: Yes, I forgot they did the big reveal a month or so back.
I ran the updates and am building with 1.x with some minor sbt fixes/headaches.
My IDE was another story.  Full upgrade was required for poor old intellij.

File: src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala (hunk 3 above)
Comment (ewheath) [reply src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala]: I can't see the other comment, but will take a shot at it.

**Comment by dspasojevic** [comment]
Nice! Thanks Eric.

**Pull request merged**